client/src/App.tsx

```diff
@@ -26,14 +25,14 @@ function App() {
       <Routes>
         <Route path='/' element={<Login/>}></Route>
         <Route path='/signup' element={<Signup/>}></Route>
+        <Route path='/new-password' element={<PasswordReset/>}></Route>
```

Comment:
Removed route protection - user will not be able to log in if they have to reset the password in the first place! (Sorry, this was included in the Protected Route reqs. My bad!)